test/libsolidity/syntaxTests/nameAndTypeResolution/010_type_conversion_for_comparison.sol

```diff
@@ -3,4 +3,4 @@ contract test {
 }
 // ----
 // Warning: (42-63): Statement has no effect.
-// Warning: (20-66): Function state mutability can be restricted to pure
+// Warning: (20-66): Function state mutability can be restricted to "pure"
```

Comment:
We have to check whether we want things like data types and visibility / pureness quoted